### PR TITLE
feat: staking APY

### LINKED ISSUE #5: APR/Y on staking page

### PR DESCRIPTION
**Pull request by 0xcamel**
Closes #5

### CODE REVIEW

**Comment by 0xcamel** [comment]
closes #5

File: ui/src/components/StakeUI.tsx [hunk 1]
```diff
@@ -50,25 +49,46 @@ export function StakeMasterJewelerUI(props: IStakeUI) {
...
 
   const loadingInformation = loadingLpToken;
-  const doing = loadingInformation || approving || staking || unstaking || claimingRewards;
+  const doing =
+    loadingInformation || approving || staking || unstaking || claimingRewards;
 
   return (
     <div className="bg-rune-edge p-4 rounded-md shadow-lg border-rune border-8 flex flex-col items-center justify-center">
       <h3 className="text-xl text-white">{props.tokenName}</h3>
+      <p className="text-gray-300">APY: {formattedAPY}%</p>
```
Comment (qdspr) [inline ui/src/components/StakeUI.tsx:77]: Loading APY needs to form part of loading state

**Review by qdspr — CHANGES_REQUESTED** [review]
Please add loading state, rest looks good

**Review by qdspr — APPROVED** [review]
LGTM

**Pull request merged**